### PR TITLE
Add Firefox Android (Fenix) 79.0

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
The brand new Firefox Android browser is now out as of last month!  This PR adds it to the browser data -- a follow-up to mirror all the data will be coming in a separate PR.

Fenix-specific release notes: https://support.mozilla.org/en-US/kb/whats-new-firefox-android-79

### CODE REVIEW

File: browsers/firefox_android.json [hunk 1]
```diff
@@ -378,7 +378,7 @@
         "60": {
           "release_date": "2018-05-09",
           "release_notes": "https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/60",
-          "status": "esr",
+          "status": "retired",
```
Comment (foolip) [inline browsers/firefox_android.json:381]: This ESR change I think would be best reviewed by @ddbeck.

**Review by foolip — APPROVED** [review]
Looks good apart from the ESR changes which I dare not review.

File: browsers/firefox_android.json (hunk 1 above)
Comment (vinyldarkscratch) [reply browsers/firefox_android.json]: I've separated this into #6627!

**Comment by foolip** [comment]
I'm suddenly in doubt if I should have merged this per https://github.com/mdn/browser-compat-data/blob/master/GOVERNANCE.md#peers...

@ddbeck to confirm, is it OK for peers to merge changes to browser releases, or only to the support data?

**Comment by vinyldarkscratch** [comment]
@foolip I've merged a number of browser data PRs and haven't heard anything negative about it, so I'd say you're all good!

**Comment by ddbeck** [comment]
@foolip Yep, @vinyldarkscratch has it right. Browser data is data too. (Adding a _new_ browser would be something else, though that would require some significant changes to the linter and such.)

**Pull request merged**